### PR TITLE
Collector docs intro

### PR DESCRIPTION
**Pull request by iusiri**

### CODE REVIEW

File: docs/pages/collectorintro.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+#Introduction
+
+Refocus makes it easy for users to understand the real-time status of their systems. To date, achieving this has required users to develop API integrations in order to push Samples into Refocus. This process had a few limitations: it required developers to write API integrations, and, at times, meant teams had to operate an extra 'always-on' service. 
+
+Collectors are designed to tackle these issues and make it easier than ever to leverage all the capabilities of Refocus!
+
+#So, what is a Collector 
+
+A Collector is an out-of-the-box application (running outside of Refocus) which collects data from different data sources and sends Samples to Refocus. It does this by leveraging user-provided config files called a “Sample Generators”.
```
Comment (iamigo) [inline docs/pages/collectorintro.md]: config files called a "Sample Generators".

should be

config files called *Sample Generators*.

File: docs/pages/collectorintro.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Most users will never have to maintain a collector (unless you choose to deploy your own), they'll just write Sample Generators.
```
Comment (iamigo) [inline docs/pages/collectorintro.md]: I don't think we can say "Most users will never have to maintain a collector". The main point to convey is that a single Collector can handle multiple Sample Generators.

File: docs/pages/collectorintro.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+##Data pipeline with collectors
+
+![Collector using SG Diagram](../assets/CollectorDataflow.jpeg)
+
+#Tell me more about Sample Generators
+
+To use a simple analogy, Collectors are akin to the *human* *body*, they do the manual heavy-lifting, but everybody (and indeed every body!) needs a brain in order to tell it what to do - the Sample Generator is that *brain*.
```
Comment (iamigo) [inline docs/pages/collectorintro.md]: heavy lifting is two words, no hyphen

File: docs/pages/collectorintro.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+
+#Introduction
+
+Refocus makes it easy for users to understand the real-time status of their systems. To date, achieving this has required users to develop API integrations in order to push Samples into Refocus. This process had a few limitations: it required developers to write API integrations, and, at times, meant teams had to operate an extra 'always-on' service. 
+
+Collectors are designed to tackle these issues and make it easier than ever to leverage all the capabilities of Refocus!
+
+#So, what is a Collector
```
Comment (iamigo) [inline docs/pages/collectorintro.md]: needs question mark

File: docs/pages/collectorintro.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+##Data pipeline with collectors
+
+![Collector using SG Diagram](../assets/CollectorDataflow.jpeg)
+
+#Tell me more about Sample Generators
+
+To use a simple analogy, Collectors are akin to the *human* *body*, they do the manual heavy-lifting, but everybody (and indeed every body!) needs a brain in order to tell it what to do - the Sample Generator is that *brain*. 
+
+We separated the *body* and the *brain* in order to modularize our system and make components interchangeable. Any Sample Generator can run on any Collector, and changes don't require system deploys making it swift and easy to get your data into Refocus.
```
Comment (iamigo) [inline docs/pages/collectorintro.md]: I don't think we can say "Any Sample Generator can run on any Collector" -- can you try rephrasing this in a way that isn't going to get us in trouble having to explain stuff to users down the line?

File: docs/pages/collectorintro.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+![Collector using SG Diagram](../assets/CollectorSGDiagram.jpeg)
+
+One Collector will run multiple Sample Generators at once - collecting and pushing Samples for each.
+
+Each Sample Generator provides a path to your target data source, a set of subjects and aspects that you want to send samples for and other contextual information used by the transform code to generate samples.
```
Comment (iamigo) [inline docs/pages/collectorintro.md]: clearer to say "other context information" (NOT "other contextual information")

File: docs/pages/collectorintro.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+We separated the *body* and the *brain* in order to modularize our system and make components interchangeable. Any Sample Generator can run on any Collector, and changes don't require system deploys making it swift and easy to get your data into Refocus.
+
+![Collector using SG Diagram](../assets/CollectorSGDiagram.jpeg)
+
+One Collector will run multiple Sample Generators at once - collecting and pushing Samples for each.
+
+Each Sample Generator provides a path to your target data source, a set of subjects and aspects that you want to send samples for and other contextual information used by the transform code to generate samples. 
+
+![Collector using SG Diagram](../assets/SGDiagram.jpeg)
+
```
Comment (iamigo) [inline docs/pages/collectorintro.md]: needs semi-colon between "with the transform code" and "instead" (not comma)

File: docs/pages/collectorintro.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+Collectors are designed to tackle these issues and make it easier than ever to leverage all the capabilities of Refocus!
+
+#So, what is a Collector 
+
+A Collector is an out-of-the-box application (running outside of Refocus) which collects data from different data sources and sends Samples to Refocus. It does this by leveraging user-provided config files called a “Sample Generators”.
+Most users will never have to maintain a collector (unless you choose to deploy your own), they'll just write Sample Generators.
+
+##Data pipeline with collectors
+
+![Collector using SG Diagram](../assets/CollectorDataflow.jpeg)
```
Comment (iamigo) [inline docs/pages/collectorintro.md]: this diagram should show multiple data sources on the right

File: docs/pages/collectorintro.md (hunk 4 above)
Comment (iusiri) [reply docs/pages/collectorintro.md]: Why? None of the titles have ending punctuation.

Comment (iusiri) [reply docs/pages/collectorintro.md]: An interesting take on not to adding a period to the end of a title applies to this too I think. https://english.stackexchange.com/questions/32884/should-i-use-a-period-at-the-end-of-a-headline

File: docs/pages/collectorintro.md (hunk 2 above)
Comment (iusiri) [reply docs/pages/collectorintro.md]: I think this is an important point to make. Most people reading this won't need to interact directly with collectors at all - they need to know that. That's the main point of the system!

File: docs/pages/collectorintro.md (hunk 7 above)
Comment (iusiri) [reply docs/pages/collectorintro.md]: No semi-colon but I removed the commas.

**Comment by iusiri** [comment]
@iamigo

File: docs/pages/collectorintro.md [hunk 9]
```diff
@@ -16,19 +15,19 @@ Most users will never have to maintain a collector (unless you choose to deploy
...
 
 One Collector will run multiple Sample Generators at once - collecting and pushing Samples for each.
 
-Each Sample Generator provides a path to your target data source, a set of subjects and aspects that you want to send samples for and other contextual information used by the transform code to generate samples. 
+Each Sample Generator provides a path to your target data source, a set of subjects and aspects that you want to send samples for and other context information used by the transform code to generate samples. 
 
 ![Collector using SG Diagram](../assets/SGDiagram.jpeg)
 
-As is illustrated, the transform code leverages user-provided value to context variables. Most users will never interact with the transform code, instead, they will select the code template that works for them (more on this later!) and provide the context variables that are required. 
+As is illustrated, the transform code leverages user-provided value to context variables. Most users will never interact with the transform code instead they will select the code template that works for them (more on this later!) and provide the context variables that are required.
```
[the remark this answers is not in the document]
Comment (iusiri) [reply docs/pages/collectorintro.md]: They aren't two different ideas/sentences. I don't see what the issue is here.

[the remark this answers is not in the document]
Comment (iamigo) [reply docs/pages/collectorintro.md]: Feel free to consult with our docs writer, Craig, if you need more help understanding the problem.

[the remark this answers is not in the document]
Comment (iusiri) [reply docs/pages/collectorintro.md]: That's condescending. Not helpful. Please approve the pr. Will be in touch with Craig.

**Comment by iamigo** [comment]
@iusiri still having CLA issue here?

**Pull request closed without merging**